src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -51,10 +53,11 @@
     @Test
     void createConsumer(VertxTestContext context) throws InterruptedException, TimeoutException, ExecutionException {
         // create consumer
-        consumerService().createConsumer(context, groupId, consumerWithEarliestReset);
+        ConsumerService consumer = consumerService().createConsumer(context, groupId, consumerWithEarliestReset);
 
         context.completeNow();
         assertTrue(context.awaitCompletion(TEST_TIMEOUT, TimeUnit.SECONDS));
+        consumer.deleteConsumer(context, groupId, name);
```

Comment:
just call `consumerService().deleteConsumer(....)` but with `deleteConsumer` on the next line :-)